src/Function.h

```diff
@@ -204,6 +210,21 @@ class Function {
     /** Check if a function has been frozen. If so, it is an error to
      * add new definitions. */
     EXPORT bool frozen() const;
+
+    /** Mark calls of this function by 'f' to be replaced with its wrapper
+     * during the lowering stage. If the string 'f' is empty, it means replace
+     * all calls to this function by all other functions (excluding itself) in
+     * the pipeline with the wrapper. See \ref Func::in for more details. */
+    EXPORT void add_wrapper(const Function &wrapper, const std::string &f = "");
+
+    /** Replace every call to function in 'substitutions' keys (including calls
+     * in the RDom's predicates) to call to function in 'substitutions' values. */
```

Comment:
In the comment, there can be more than one RDom (one for each update) correct? Tweak the comment accordingly?